### PR TITLE
MNP-162 reduce similar dockerfiles

### PR DESCRIPTION
**Pull request by Glen-Moonpig**
Now passing in the base image version as a docker build argument instead of creating duplicate dockerfiles.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -84,12 +78,13 @@ AWS CI build:
 Dotnet21 release build:
   stage: Build
   variables:
+    MNPG_BASE_IMAGE_VERSION: 2.1-sdk-alpine
```
Comment (jaknor) [inline .gitlab-ci.yml]: Should we share the base image between the release job and CI build job? They will always be the same wont they?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -146,12 +142,13 @@ Dotnet21 CI build:
 Dotnet22 release build:
   stage: Build
   variables:
+    MNPG_BASE_IMAGE_VERSION: 2.2-sdk-alpine
```
Comment (jaknor) [inline .gitlab-ci.yml]: Should we share the base image between the release job and CI build job? They will always be the same wont they?

File: .gitlab-ci.yml (hunk 1 above)
Comment (Glen-Moonpig) [reply .gitlab-ci.yml]: I have created a 'pipeline-level' variable

File: .gitlab-ci.yml (hunk 2 above)
Comment (Glen-Moonpig) [reply .gitlab-ci.yml]: I have created a 'pipeline-level' variable

**Pull request merged**